solutions/usaco-717.mdx

```diff
@@ -2,14 +2,93 @@
 id: usaco-717
 source: USACO Gold 2017 December
 title: Why Did the Cow Cross the Road
-author: Qi Wang
+author: Qi Wang, Jeffrey Zhang
 ---
 
 [Official Analysis](http://www.usaco.org/current/data/sol_visitfj_gold_feb17.html)
 
+## Solution
+
+Instead of treating this problem with grids, let us consider from a more graphical perspective with nodes and edges.
+
+We first observe that because Bessie only feasts on grass tiles every 3 moves and the time to travel between adjacent tiles is a constant $T$. The two intermediary tiles Bessie used to travel does not actually matter. So we want to form a direct edge between each tile that are exactly 3 manhattan distances apart, with an weight of 
```

Comment:
```suggestion
We first observe that the two intermediary tiles Bessie used to travel do not actually matter because Bessie only feasts on grass tiles every 3 moves and the time to travel between adjacent tiles is a constant $T$. So we want to form a direct edge between each tile that is exactly 3 manhattan distances apart, with a weight of 
```